tests/unit/amo/components/TestAddon.js

```diff
@@ -1293,25 +1128,9 @@ describe(__filename, () => {
       const root = renderMoreAddons({ addon, addonsByAuthors });
 
       expect(root).not.toHaveClassName('.AddonDescription-more-addons--theme');
-      expect(root).toHaveProp('addons', addonsByAuthors.map((addonByAuthor) => (
-        createInternalAddon(addonByAuthor)
-      )));
-    });
-
-    it('indicates when other add-ons are themes', () => {
```

Comment:
I think we still need a test case that verifies that the `MoreAddonsByAuthorsCard` receives the correct properties when the add-on is a theme.

Reply:
Ah right, fair enough. I'll add that since they're displayed in different places... actually maybe I'll just put it in a function so we don't have to test it twice.

Reply:
Actually enough things are different I'd rather just have another test, but will do.